### ISSUE TITLE
LDAP: Nested Groups should be active by default

### ISSUE THREAD

**Issue by GitBaer** [YunoHost-Apps/nextcloud_ynh#529]
Nested groups should be active by default so Yunohost-groups will be synchronised correctly:

![grafik](https://user-images.githubusercontent.com/49997071/206680983-ba571e18-1338-4312-9695-e5a63fee3341.png)

**Comment by tituspijean** [comment]
Hello @GitBaer, can you describe how the groups are not correctly synchronized in the current configuration?

**Comment by zamentur** [comment]
In french version, i have this help:
> Groupes imbriqués: Si activé, les groupes contenant d'autres groupes sont pris en charge (fonctionne uniquement si l'attribut membre du groupe contient des DN).

So i guess yunohost is not concerned by this, cause we don't have groups inside other groups

@GitBaer could you give more details ?

**Comment by MCMic** [comment]
Yeah I do not see why nested groups would be needed for Yunohost, it’s also a common source of problems for Nextcloud so I advise against it.
